File: csharp/gigasecond/Gigasecond.cs
using System;

public static class Gigasecond
{
    public static DateTime Add(DateTime moment)
    {
        double gigaSecond = Math.Pow(10.0, 9.0);
        Console.WriteLine(gigaSecond);
        DateTime gigaMoment = moment.AddSeconds(gigaSecond);
        return gigaMoment;
    }
}

File: javascript/gigasecond/gigasecond.js
//
// This is only a SKELETON file for the 'Gigasecond' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const gigasecond = (date) => {
  var gs = new Date(date.getTime() + Math.pow(10,12));
  return gs; 
};
